backend/src/schema/types/type/EmailAddress.gql

```diff
@@ -19,5 +18,6 @@ type Mutation {
     avatarUpload: Upload
     locationName: String
     about: String
+    termsAndConditionsAgreedVersion: String!
```

Comment:
```suggestion
    termsAndConditionsAgreedVersion: String
```
@ogerly I think you can allow this to be `null`. If we import the users, they will have `termsAndConditionsAgreedVersion == null`. So that's a valid situation where this is null. The check in the nuxtjs middleware should still work as expected.